pkg/apis/kops/instancegroup.go

```diff
@@ -178,6 +181,16 @@ const (
 // SpotAllocationStrategies is a collection of supported strategies
 var SpotAllocationStrategies = []string{SpotAllocationStrategyLowestPrices, SpotAllocationStrategyDiversified, SpotAllocationStrategyCapacityOptimized}
 
+// InstanceMetadataOptions holds the fields required to configure instance metadata in an instance.
+// This currently holds fields to require EC2 Instance Metadata in EC2
+type InstanceMetadataOptions struct {
+	// HTTPPutResponseHopLimit is the max no of hops made by the PUT request which fetches the token(default '1')
+	HTTPPutResponseHopLimit *int64 `json:"httpPutResponseHopLimit,omitempty"`
+	// HTTPTokens can be either 'required' or 'optional'. If value is 'required', then EC2 IMDv2 will be enabled and
+	// EC2 IMDv1 will be disabled. If it is 'optional', both v1 and v2 will be enabled (default 'optional')
```

Comment:
From: https://docs.aws.amazon.com/sdk-for-go/api/service/autoscaling/#InstanceMetadataOptions
```suggestion
	// HTTPTokens is the state of token usage for the instance metadata requests.
	// If the parameter is not specified in the request, the default state is "optional".
```